catalog/models.py

```diff
@@ -119,42 +122,32 @@ def active(self):
 class ProductManager(models.Manager.from_queryset(ProductQuerySet)):
     """Get all products of given category by Category's id or instance."""
 
-    def get_by_category(self, category: models.Model, ordering: [str]=None) -> models.QuerySet:
+    def get_by_category(
+        self, category: models.Model, ordering: [str]=None
```

Comment:
`[str]=None -> [str] = None` and below too

Reply:
@artemiy312 , we use no spaces around default arg operator. I think type annotations is not good reason for special tactics.
Let's follow pep8's recommendation (item 4)
https://www.python.org/dev/peps/pep-0008/#other-recommendations

Reply:
Moved this discussion to https://github.com/fidals/refarm-site/issues/191